README.md

```diff
@@ -106,30 +102,33 @@ Somewhere in your service that's about to make an outgoing call:
     def before_http_request(request, current_span_extractor):
         op = request.operation
         parent_span = current_span_extractor()
-        if parent_span is None:
-            span = opentracing.tracer.start_trace(operation_name=op)
-        else:
-            span = parent_span.start_child(operation_name=op)
+        outbound_span = opentracing.tracer.start_span(
+            operation_name=op,
+            parent=parent_span
+        )
     
-        span.set_tag('server.http.url', request.full_url)
+        outbound_span.set_tag('server.http.url', request.full_url)
```

Comment:
@adriancole pointed out elsewhere, this should be `http.url` - filed https://github.com/opentracing/opentracing-python/issues/18